packages/@react-spectrum/s2/stories/ActionButton.stories.tsx

```diff
@@ -203,14 +204,16 @@ export const Avatars: Story = {
 
 export const NotificationBadges: Story = {
   render: (args) => {
+    let badgeValue = 10;
+    let formattedValue = new NumberFormatter('en-US').format(badgeValue);
```

Comment:
we should use https://react-spectrum.adobe.com/react-aria/useNumberFormatter.html#usenumberformatter so it picks up the locale automatically